frontend/src/main/scala/bloop/exec/JvmProcessForker.scala

```diff
@@ -71,6 +72,9 @@ trait JvmProcessForker {
 }
 
 object JvmProcessForker {
+  // Windows max cmd line length is 32767, which seems to be the least of the common shells.
+  val classpathCharLimit: Int = 30000
```

Comment:
The char limit is associated with the length of the command that we use to sysprocess, not the classpath char itself. What about we fix that and we make it private?

```suggestion
  private val processCmdCharLimit: Int = 30000
```
  
Also, what about moving this definition as close to the use site as possible?

Reply:
done